package.json

```diff
@@ -33,6 +33,7 @@
     "dotenv": "^16.0.1",
     "eslint": "^8.21.0",
     "eslint-config-prettier": "^8.5.0",
+    "esm": "^3.2.25",
```

Comment:
what is this for?

Reply:
It's to run the partial manifest script. I uninstalled it by mistake and then re-installed as a dev dependency

Reply:
I don't see the uninstall I only see this in this PR